### ISSUE TITLE
value-keyword-case shouldn't throw an error for: optimizeSpeed, optimizeLegibility, geometricPrecision 

### ISSUE THREAD

**Issue by pawelgrzybek** [stylelint/stylelint#1089]
Whenever I use one of these values: optimizeSpeed, optimizeLegibility, geometricPrecision, stylelint throws an error.

It is related with "value-keyword-case" rule.
I'm on 6.0.1 version.
Run via CLI.
I'm my opinion this rule should treat these values as exceptions.

```shell
40:19  ✖  Expected "optimizeLegibility" to be "optimizelegibility"   value-keyword-case
```

**Comment by jeddy3** [comment]
Oh, the [good ol' SVG properties](https://developer.mozilla.org/en-US/docs/Web/CSS/text-rendering) rearing their ugly heads. 

> I'm my opinion this rule should treat these values as exceptions.

Agreed.

@pawelgrzybek If you'd like to contribute a fix for this, you can follow [this similar addition](https://github.com/stylelint/stylelint/pull/1083) to `function-name-case`-- which added exceptions for the likes of `translateX()`.

**Comment by pawelgrzybek** [comment]
No worries, I will do my best to contribute. Thanks @jeddy3

**Comment by evilebottnawi** [comment]
@pawelgrzybek Would you like start PR?

**Comment by evilebottnawi** [comment]
Merged. Close.

**Comment by ChazUK** [comment]
@alexander-akait this has since been [removed](https://github.com/stylelint/stylelint/blob/main/lib/rules/value-keyword-case/index.js). The camel case is still the format in which `text-rendering` receives it's assigned value. https://developer.mozilla.org/en-US/docs/Web/CSS/text-rendering

**Comment by jeddy3** [comment]
@ChazUK Thanks for chiming in.

> The camel case is still the format in which text-rendering receives it's assigned value.

Is the lowercase keyword not working for you? If not, it'd be great to know what browsers and versions are affected.

The [specs seems to be](https://www.w3.org/TR/css-color-4/#currentcolor-color) moving towards lowercase keywords even for these SVG keywords, which is why the default behaviour has changed. 

If you want to stick with camel case, you can use the [`camelCaseSvgKeywords` option](https://stylelint.io/user-guide/rules/list/value-keyword-case/#camelcasesvgkeywords-true--false-default-false).

**Comment by jeffnyman** [comment]
I'm curious why this was closed as this still seems to happen for `optimizeSpeed`. Are we saying styleline is correct and the specs are wrong? I'm using version 15.2.0.  The only way I've found to get rid of the issue is to disable the `value-keyword-case` for the section that contains any `text-rendering` elements.
